### PR TITLE
adding font size & wight

### PR DESCRIPTION
**Pull request by Akram-Sa**

### CODE REVIEW

File: frontend/src/utility.scss [hunk 1]
```diff
@@ -20,6 +22,25 @@
...
+  --fs-extra-small: 0.75rem;
+  --fs-small: 0.875rem;
+  --fs-regular: 1rem;
+  --fs-medium: 1.125rem;
+  --f-large: 1.25rem;
+  --fs-extra-large: 1.5rem;
+  --fs-2x-extra-large: 1.875rem;
+  --fs-3x-extra-large: 2.25rem;
+
+  --fw: 100;
```
[the remark this answers is not in the document]
Comment (Akram-Sa) [reply frontend/src/utility.scss:35]: please check updates

[the remark this answers is not in the document]
Comment (Akram-Sa) [reply frontend/src/utility.scss:35]: please check updates
#screenshots not supporting!

[the remark this answers is not in the document]
Comment (akibur-r) [reply frontend/src/utility.scss:35]: Add the screenshots in this comment section, along with the corresponding codes.

[the remark this answers is not in the document]
Comment (akibur-r) [reply frontend/src/utility.scss:35]: Also, restore the App.jsx file to its original form. We want the project site running in the App

[the remark this answers is not in the document]
Comment (Akram-Sa) [reply frontend/src/utility.scss:35]: testinng
Here it is

[the remark this answers is not in the document]
Comment (Akram-Sa) [reply frontend/src/utility.scss:35]: its jsx code:
t code

[the remark this answers is not in the document]
Comment (Akram-Sa) [reply frontend/src/utility.scss:35]: and SCSS code:

[the remark this answers is not in the document]
Comment (Akram-Sa) [reply frontend/src/utility.scss:35]: SCSS code:
t code 2

**Pull request closed without merging**